src/views/name.njk

```diff
@@ -94,7 +94,9 @@
         </fieldset>
       </div>
 
-      {% include "includes/save-and-continue-button.njk" %}
+      <button class="govuk-button" data-module="govuk-button" id="submit" onclick="_paq.push(['trackEvent', 'Which limited partnership name ending would you prefer to go on the public record?', document.querySelector('input[name=name_ending]:checked').nextElementSibling.innerHTML])">
```

Comment:
Maybe amount of template code can be reduced here if i18n text used.

Reply:
tried the i18n text, but the text will change depending on what type of LP it is, eg. SLP etc. I'm guessing that there will be some reporting tool perhaps looking for a particular string, so decided to use a generic name to indicate the name-ending value.
Also switched to using the govuk button, i didn't think it would work with onClick but it does.